stack.yaml

```diff
@@ -1,10 +1,11 @@
-resolver: lts-13.14
+resolver:
+  url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/17/5.yaml
```

Comment:
Why not?

```suggestion
resolver: lts-17.5
```

Reply:
That's much better, done in latest commit!